plugins/github-pull-requests-board/src/components/icons/ArchivedRepo/ArchivedRepo.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { FunctionComponent } from 'react';
+
+const ArchivedRepo: FunctionComponent = () => (
```

Comment:
We [avoid the use of FC](https://backstage.io/docs/architecture-decisions/adrs-adr006) variants, you can just skip the type entirely here.

Reply:
Another possibility would perhaps be to pick one of the `@material-ui/icons` icons. https://v4.mui.com/components/material-icons/